customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -78,3 +78,5 @@ function check_ib_bw_gdr(){
    wait  ${pid_array[@]}
    return 0
 }
+
+# check_ib_bw_gdr 170.0 nd96asr_v4
```

Comment:
Do we really need it? And it seems changed to 175 in [conf/nd96asr_v4.conf](https://github.com/Azure/azurehpc-health-checks/pull/50/files#diff-b4e63cc69f103889afc329719236165bb6d26b191bf18b9e0e9087c0e6927a13)

Reply:
Good catch it was left over from testing. Let me remove

Reply:
addressed